firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -48,15 +48,18 @@ bool isValidIM21Id(String input) {
 }
 
 int toggle_debugging_publishes(String command){
-
   //default to invalid input
   int returnFlag = -1;
 
   //string.toInt() returns 0 if it fails, so can't distinguish between user 
   //entering 0 vs entering bad input. So convert to char and use ascii table
   const char* holder = command.c_str();
 
-  if(*(holder+1) != 0){
+  // if e, echo whether debug publishes are on
+  if (*holder == 'e') {
+    returnFlag = stateMachineDebugFlag;
```

Comment:
How does this get translated from a boolean into an int?

Reply:
I made a change explicitly casting stateMachineDebugFlag to an int so that the cast is explicit and not implicit